src/scss/tools/mixins/_map_deep_get.scss

```diff
@@ -0,0 +1,12 @@
+/// Map deep get
+/// @author Hugo Giraudel
+/// @access public
+/// @param {Map} $map - Map
+/// @param {Arglist} $keys - Key chain
+/// @return {*} - Desired value
```

Comment:
@eduardojmatos what do you think about we adjust this documentation after we create others components